data/debian-distro-info.json

```diff
@@ -0,0 +1,19 @@
+{
+  "7": "wheezy",
+  "8": "jessie",
+  "9": "stretch",
+  "10": "buster",
+  "11": "bullseye",
+  "12": "bookworm",
+  "1.1": "buzz",
+  "1.2": "rex",
+  "1.3": "bo",
+  "2.0": "hamm",
+  "2.1": "slink",
+  "2.2": "potato",
+  "3.0": "woody",
+  "3.1": "sarge",
+  "4.0": "etch",
+  "5.0": "lenny",
+  "6.0": "squeeze"
```

Comment:
Was this file intended to be added to this PR?

Reply:
IMO it shouldn't be. I know I commented that we should cover Debian release names, but it should be a separate PR. There is some added complexity with Debian and it's naming of `stable`, `oldstable`, `oldoldstable`, etc.

Reply:
removed the above